### DISCUSSION TITLE
关于视频缓冲

### DISCUSSION THREAD

**Discussion by Rcrwrate** [ywmoyue/biliuwp-lite#826] [feature]
### 问题描述

网络卡顿与视频预载

### 描述解决方案

能否像potplayer一样，只要我在视频详情页**开始播放之后**就一路完整的预载到结束，除非我关闭了该页面

这个在网络卡顿的时候应该很有用

或者允许调整视频预加载的长度/(时间/秒)也可以

### 代码贡献

否

**Comment by ywmoyue** [comment]
暂时不能，后续有空再看看
